ahelal/ansible-sonatype_nexus
# Nexus improvments

Hi,

I have a few improvments for your role :+1: 
- Add travis CI build (test)
- Add automatic documentation
- Check / fix nexus ownership

Do not hesitate to tell me if you need me to remove or something else.

ALban
